components/favorite-timetables-list.vue

```diff
@@ -8,7 +8,7 @@
         icon
         flat
         @click="favoriteTimetableDialogOpen = true">
-        <v-icon>add</v-icon>
+        <v-icon>menu</v-icon>
```

Comment:
Das bringt die Intention besser rüber 👍 
Jetzt sollte es noch nach links (als primäre Aktion) oder nach rechts (als sekundäre Aktion), Menüs sind meistens rechts (außer auf iOS)

Reply:
was meinst du mit nach links und nach rechts

Reply:
like this?
<img src=https://user-images.githubusercontent.com/15379000/48833044-db8c5400-ed7a-11e8-9967-d2f2ba7de3f2.png height=300 />